sdk/index.rst

```diff
@@ -0,0 +1,14 @@
+.. PPMS-PDFD documentation master file, created by
+   sphinx-quickstart on Tue Dec 12 14:22:39 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Piwik PRO SDK documentation!
+=======================================
+
+.. toctree::
+   :maxdepth: 2
+   :name: test
```

Comment:
Im still not sure how this `name` is used. I think you can skip it unless you need it.